build.sh

```diff
@@ -117,11 +142,11 @@ case "$1" in
 		;;
 esac
 
-while getopts "bwd" opt; do
+while getopts "b:wd" opt; do
   case $opt in
     b)
-        echo "Build docs"
-        build 'emerald-cli'
+        echo "Build docs: $OPTARG"
```

Comment:
It supposed to be called for all projects, one by one, like:
```
build 'emerald-cli'
build 'emerald-js'
build 'sputnik-vm-spec'
build 'go-ethereum'
build 'json-rpc'
```
The goal is to make whole documentation across all projects in one call, and then publish it on website